README.md

```diff
@@ -23,8 +23,8 @@ FLASK_APP=plasma_cash/child_chain FLASK_ENV=develment flask run --port=8546
 Client:
 ```
 python
->>> from plasma_cash.client.client import Client
->>> c = Client()
+>>> from plasma_cash.dependency_config import container
+>>> c = container.get_client()
 >>> c.deposit(100, '0xb83e232458a092696be9717045d9a605fb0fec2b')
```

Comment:
This should be changed to `c.deposit(100, '0xb83e232458a092696be9717045d9a605fb0fec2b', '0x0000000000000000000000000000000000000000')` ?

Reply:
yes, my bad. Thank for catching this out!